server/src/main/java/org/apache/druid/server/coordination/ZkCoordinator.java

```diff
@@ -54,21 +56,25 @@
 
   private volatile PathChildrenCache loadQueueCache;
   private volatile boolean started = false;
+  private final ExecutorService segmentLoadUnloadService;
 
   @Inject
   public ZkCoordinator(
       SegmentLoadDropHandler loadDropHandler,
       ObjectMapper jsonMapper,
       ZkPathsConfig zkPaths,
       DruidServerMetadata me,
-      CuratorFramework curator
+      CuratorFramework curator,
+      SegmentLoaderConfig config
   )
   {
     this.dataSegmentChangeHandler = loadDropHandler;
     this.jsonMapper = jsonMapper;
     this.zkPaths = zkPaths;
     this.me = me;
     this.curator = curator;
+    this.segmentLoadUnloadService = Execs.multiThreaded(
+        config.getNumLoadingThreads(), "ZKCoordinator--%d");
```

Comment:
Not according to Druid code style

Reply:
I wish Intellij or Eclipse would just format this correctly. Working on multiple projects, it gets difficult to remember coding styles of individual projects. And doesn't seem like checkstyle complained about this either.